iOS/Projects/Features/Record/Sources/ExerciseSelectScene/View/ExerciseSelectViewController.swift

```diff
@@ -0,0 +1,109 @@
+    collectionView.register(ExerciseCardCell.self, forCellWithReuseIdentifier: ExerciseCardCell.identifier)
+
+    collectionView.translatesAutoresizingMaskIntoConstraints = false
+    return collectionView
+  }()
+
+  private let nextButton: UIButton = {
+    let button = UIButton()
+    button.configurationUpdateHandler = UIButton.Configuration.main(label: "다음")
+
+    button.translatesAutoresizingMaskIntoConstraints = false
+    return button
+  }()
+}
+
+private extension ExerciseSelectViewController {
+  func makeCollectionViewLayout() -> UICollectionViewLayout {
+    let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(0.5), heightDimension: .fractionalHeight(1))
+
+    let item = NSCollectionLayoutItem(layoutSize: itemSize)
+    item.contentInsets = .init(
+      top: Const.CellInset,
+      leading: Const.CellInset,
+      bottom: Const.CellInset,
+      trailing: Const.CellInset
+    )
+
+    let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                           heightDimension: .fractionalWidth(0.55))
+    let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [item])
+
+    let section = NSCollectionLayoutSection(group: group)
+
+    return UICollectionViewCompositionalLayout(section: section)
+  }
+
+  func setupConstraints() {
+    let safeArea = view.safeAreaLayoutGuide
+
+    view.addSubview(exerciseSelectDescriptionLabel)
+    exerciseSelectDescriptionLabel.topAnchor.constraint(equalTo: safeArea.topAnchor).isActive = true
+    exerciseSelectDescriptionLabel.leadingAnchor
+      .constraint(equalTo: safeArea.leadingAnchor, constant: ConstraintsGuideLine.value).isActive = true
+    exerciseSelectDescriptionLabel.trailingAnchor
+      .constraint(equalTo: safeArea.trailingAnchor, constant: -ConstraintsGuideLine.value).isActive = true
+
+    view.addSubview(exerciseCardCollectionView)
+    exerciseCardCollectionView.topAnchor
+      .constraint(equalTo: exerciseSelectDescriptionLabel.bottomAnchor, constant: 12).isActive = true
+    exerciseCardCollectionView.leadingAnchor
+      .constraint(equalTo: safeArea.leadingAnchor, constant: ConstraintsGuideLine.value).isActive = true
+    exerciseCardCollectionView.trailingAnchor
+      .constraint(equalTo: safeArea.trailingAnchor, constant: -ConstraintsGuideLine.value).isActive = true
+    exerciseCardCollectionView.bottomAnchor
+      .constraint(equalTo: view.bottomAnchor, constant: -15).isActive = true
+
+    view.addSubview(nextButton)
+    nextButton.leadingAnchor
+      .constraint(equalTo: safeArea.leadingAnchor, constant: ConstraintsGuideLine.value).isActive = true
+    nextButton.trailingAnchor
+      .constraint(equalTo: safeArea.trailingAnchor, constant: -ConstraintsGuideLine.value).isActive = true
+    nextButton.bottomAnchor
+      .constraint(equalTo: safeArea.bottomAnchor, constant: -28).isActive = true
+  }
+
+  enum Const {
+    static let CellInset: CGFloat = 5
+  }
```

Comment:
p2: 변수명이 Pascal Case로 되어있습니다! SwiftFormat이 안잡아주나보네요 흠 ?
```suggestion
  enum Const {
    static let cellInset: CGFloat = 5
  }
```

Reply:
상수 관리를 PascalCase로 해서 상수인것을 자명하게 하는것도 방법이라는 말을 들어서 그렇게 사용한 것 같네요 ㅎㅎ

문제점은, 코드에서 PascalCase와 SnakeCase가 혼용해서 쓰인 것 같습니다. 

따라서 상수 부분에 대해서 종표님과 이야기 하면 좋아보입니다.